FAQ.md

```diff
@@ -85,11 +85,29 @@ Here are some suggestions for when you could use `dep` or `go get`:
 -[@sdboyer in #376](https://github.com/golang/dep/issues/376#issuecomment-294045873)
 
 ## Why is `dep` ignoring a version constraint in the manifest?
-Only your project's directly imported dependencies are affected by a `dependencies` entry
+Only your project's directly imported dependencies are affected by a `constraint` entry
 in the manifest. Transitive dependencies are unaffected.
 
 Use an `overrides` entry for transitive dependencies.
 
+To pin a version of direct dependency in manifest, prefix the version with `=`.
+For example:
+```
+[[constraint]]
+  name = "github.com/pkg/errors"
+  version = "=0.8.0"
+```
+
+By default, when you specify a version without an operator, such as `~` or `=`,
```

Comment:
I feel like it makes more sense to switch the order of these 2 sections. How the default case is handled should come first and then how to pinpoint an exact version.